docs/versioned_docs/version-1.0/20-usage/35-advanced-yaml-syntax.md

```diff
@@ -88,3 +88,8 @@ steps:
      - echo echo from second step
      - <<: *post_cmds
 ```
+
+## References
+
+- [Official specification](https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases)
+- [Cheatsheet](https://learnxinyminutes.com/docs/yaml)
```

Comment:
I'm pretty sure there are a lot of good YAML resources on the internet, but I'm not sure if we should start linking to them. IMO the official spec is a good resource in addition to the woodpecker documentation and anyone can search further on the internet.

Reply:
Ok good point 👍